Core/include/Acts/Geometry/Volume.hpp

```diff
@@ -112,11 +111,10 @@ class Volume : public GeometryObject {
   /// - as default the center is given, but may be overloaded
   ///
   /// @param gctx The current geometry context object, e.g. alignment
-  /// @param bValue is the binning value schema
-  ///
+  /// @param aDir is the acis direction for the reference position
```

Comment:
```suggestion
  /// @param aDir is the axis direction for the reference position
```

Reply:
Fixed, wonder why the spelling didn't catch that ?